packages/lodestar/src/network/gossip/gossip.ts

```diff
@@ -56,14 +56,16 @@ export class Gossip extends (EventEmitter as {new (): GossipEventEmitter}) imple
     // need to improve Gossipsub type to implement EventEmitter to avoid this cast
     this.pubsub =
       pubsub ||
-      ((new LodestarGossipsub(config, validator, this.logger, libp2p.peerId, libp2p.registrar, {
+      // @ts-ignore
```

Comment:
Could you find a way to avoid using ts-ignore here?

Reply:
oh, just saw this comment that explains why it's here
https://github.com/ChainSafe/lodestar/pull/1541#discussion_r499922451